### PR TITLE
Added link to UML website and provided example

### LINKED ISSUE #26: 61-UseCasePic: Add link to UML website and provide more examples
The document briefly explains what a UML is but it would be more helpful to provide more information on UML by adding a link to the UML website: https://www.uml.org/
Additionally, it would be helpful to provide more examples on how use case diagrams are used in software development. By giving examples on the benefit of use case diagrams in real work settings, students will understand the value of it.

### PR DESCRIPTION
**Pull request by Amr-mah**
fixes #26

### CODE REVIEW

File: docs/labs/61-UseCasePic.md [hunk 1]
```diff
@@ -25,6 +26,16 @@ that everyone can follow when designing software,
 similar to how we use certain words or names to evoke certain meanings,
 like the name Karen for example.
 
+For a more concrete example of how UML is used in software development,
+consider the following use case.
+Imagine you are a software developer working on a mobile banking app.
+To ensure that the application is designed and built to meet the needs of the end-users,
+you and your team would create a use case diagram using UML to outline all interactions
+like checking account balances to pay bills.
+Each interaction is a use case. The diagram will identify external systems tha app needs,
```
Comment (onzfonz) [inline docs/labs/61-UseCasePic.md:35]:
```suggestion
Each interaction is a use case. The diagram will identify external systems that the app needs,
```

File: docs/labs/61-UseCasePic.md [hunk 2]
```diff
@@ -25,6 +26,16 @@ that everyone can follow when designing software,
...
+like payment gateways or databases.
+By using UML, you get a high level view and ensure the app meets user needs.
```
Comment (onzfonz) [inline docs/labs/61-UseCasePic.md:37]:
```suggestion
By using UML, you get a high-level view and ensure the app meets user needs.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Also did you get this example from somewhere? If so we need to cite

**Comment by Amr-mah** [comment]
> Also did you get this example from somewhere? If so we need to cite
I didnt get the example from somewhere

File: docs/labs/61-UseCasePic.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ which is a part of **UML** which stands for *Unified Modeling Language*.
 
 UML is good for getting a high-level understanding of a piece of software
 and how different components interact with one another.
+If you want to learn more about UML, you can visit the official UML website here: [UML](https://www.uml.org/).
```
Comment (onzfonz) [inline docs/labs/61-UseCasePic.md:19]:
```suggestion
If you want to learn more about UML, you can visit the [official UML website](https://www.uml.org/).
```

File: docs/labs/61-UseCasePic.md [hunk 4]
```diff
@@ -25,6 +26,16 @@ that everyone can follow when designing software,
 similar to how we use certain words or names to evoke certain meanings,
 like the name Karen for example.
 
+For a more concrete example of how UML is used in software development,
+consider the following use case.
+Imagine you are a software developer working on a mobile banking app.
```
Comment (onzfonz) [inline docs/labs/61-UseCasePic.md:31]:
```suggestion

> Imagine you are a software developer working on a mobile banking app.
```

File: docs/labs/61-UseCasePic.md [hunk 5]
```diff
@@ -25,6 +26,16 @@ that everyone can follow when designing software,
...
+To ensure that the application is designed and built to meet the needs of the end-users,
+you and your team would create a use-case diagram using UML to outline all the major interactions,
+like checking account balances to pay bills.
+Each interaction is a use case. The diagram will identify external systems that the app needs,
```
Comment (onzfonz) [inline docs/labs/61-UseCasePic.md:35]:
```suggestion

Each interaction is a use case. The diagram will identify external systems that the app needs,
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
just do these three changes via the commit suggestion on the website and we'll be good

**Review by onzfonz — APPROVED** [review]

**Pull request merged**